docs-api/frontity-packages/collections-packages/hooks/infinite-scroll-hooks.md

```diff
@@ -204,7 +149,7 @@ It accepts an optional object with the following props:
 > NOTE: The IntersectionOptions type refers to the type of the the parameters
 > received by the [`useInView` hook](https://api.frontity.org/frontity-packages/collections-packages/hooks#parameters).
 
-#### Return value
+### Return value
 
 The output of these hooks is pretty similar to the previous one's:
 
```

Comment:
Do the `isFetching` and `isError` properties refer to posts or **pages of posts**?

Reply:
@DAreRodz Can you also clarify the purpose of these properties a bit more?

Reply:
In this case, `isFetching` refers to **pages of posts**, as posts are not fetched individually. [(link)](https://github.com/frontity/frontity/blob/b1a7e55015b368ea1819a2e0c745edb27eee4df1/packages/hooks/use-post-type-infinite-scroll/index.tsx#L203-L204)

However, `isError` refers both to **posts** and **pages of posts**: it would be `true` if the last post or the last page fetched have populated an error data object. [(link)](https://github.com/frontity/frontity/blob/b1a7e55015b368ea1819a2e0c745edb27eee4df1/packages/hooks/use-post-type-infinite-scroll/index.tsx#L205)